ts/boot/configureStoreAndPersistor.ts

```diff
@@ -92,11 +92,11 @@ const sagaMiddleware = createSagaMiddleware();
  * @https://github.com/react-navigation/react-navigation/issues/3438.
  */
 const navigation = createReactNavigationReduxMiddleware(
+  // This is a selector to get the navigation state from the global state
```

Comment:
is this change necessary? looks like just a reordering of the rows

Reply:
It is a reordering of the params
oldversion -> key, selector
newversion -> selector, key